albam/engines/mtfw/structs/mod-156.ksy

```diff
@@ -270,38 +249,36 @@ types:
     seq:
       - {id: idx_group, type: u2}
       - {id: idx_material, type: u2}
-      - {id: constant, type: u1}
+      - {id: disp, type: u1}
       - {id: level_of_detail, type: u1}
-      - {id: z_buffer_order, type: u1}
-      - {id: vertex_format, type: u1}
+      - {id: alpha_priority, type: u1} # alphapri
+      - {id: max_bones_per_vertex, type: u1} # weight_num
       - {id: vertex_stride, type: u1}
       - {id: vertex_stride_2, type: u1}
-      - {id: unk_03, type: u1}
-      - {id: unk_flag_01, type: b1}
-      - {id: unk_flag_02, type: b1}
-      - {id: unk_flag_03, type: b1}
-      - {id: unk_flag_04, type: b1}
-      - {id: unk_flag_05, type: b1}
-      - {id: use_cast_shadows, type: b1}
-      - {id: use_receive_shadows, type: b1}
-      - {id: unk_flag_08, type: b1}
+      - {id: connective, type: u1}
+      - {id: shape, type: b1}
+      - {id: env, type: b1}
+      - {id: refrect, type: b1}
+      - {id: reserved2_flag_1, type: b1} # probably unused b2 value but need to check
+      - {id: reserved2_flag_2, type: b1} # probably unused b2 value but need to check
+      - {id: shadow_cast, type: b1}
+      - {id: shadow_receive, type: b1}
+      - {id: sort, type: b1}
       - {id: num_vertices, type: u2}
       - {id: vertex_position_end, type: u2}
       - {id: vertex_position_2, type: u4}
       - {id: vertex_offset, type: u4}
       - {id: vertex_offset_2, type: u4} # second vertex buffer offset
-      - {id: face_position, type: u4}
+      - {id: face_position, type: u4} # index_ofs
       - {id: num_indices, type: u4}
-      - {id: face_offset, type: u4}
-      - {id: unk_06, type: u1}
-      - {id: unk_07, type: u1}
-      - {id: vertex_position, type: u2}
-      - {id: num_unique_bone_ids, type: u1}
-      - {id: idx_bone_palette, type: u1}
-      - {id: unk_08, type: u1}
-      - {id: unk_09, type: u1}
-      - {id: unk_10, type: u2}
-      - {id: unk_11, type: u2}
+      - {id: face_offset, type: u4} # index_base
+      - {id: vdeclbase, type: u1}
+      - {id: vdecl, type: u1}
+      - {id: vertex_position, type: u2} # min_index
+      - {id: num_weight_bounds, type: u1} # probably num of OABB boundng volumes
```

Comment:
Nit: I think this was confirmed long ago, just with a bad name (there was a TODO to rename it), so the comment can be removed